server/src/main/kotlin/org/itmodreamteam/myrest/server/service/restaurant/RestaurantService.kt

```diff
@@ -0,0 +1,15 @@
+package org.itmodreamteam.myrest.server.service.restaurant
+
+import org.itmodreamteam.myrest.server.model.restaurant.Restaurant
+import org.itmodreamteam.myrest.shared.restaurant.RestaurantRegistrationInfo
+import org.itmodreamteam.myrest.shared.restaurant.RestaurantUpdateInfo
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.Pageable
+
+interface RestaurantService {
+
+    fun register(newRestaurant: RestaurantRegistrationInfo)
+    fun update(restaurantUpdateInfo: RestaurantUpdateInfo)
+    fun getById(id: Long) : Restaurant
```

Comment:
We don't want to return Restaurant entity, but we rather want to return a DTO (RestaurantInfo), placed into shared library so that the clients can reuse it